spec/factories/teams.rb

```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
+  factory :random_team do
```

Comment:
I think you should delete this and modify your `:team` factory. My suggestion would be to use a [sequence](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#sequences) so they are unique. Your team names and cities will look something like `Patriots 17` but that's fine.